csv2rdf/csv2rdf_single_subject.py

```diff
@@ -0,0 +1,75 @@
+import csv
+import validators
+import sys
+import json
+import os
+from rdflib import Graph, URIRef, Literal
```

Comment:
This script requires a third-party library. Is that documented either in a Readme, a requirements.txt or a pyproject.toml file?

Reply:
It's in pyproject.toml.

Reply:
Then the `pyproject.toml` should be in this PR.

Reply:
The pyproject.toml is in the main branch. It's not changed.

Reply:
These lines are moved directly from another file, they should also be the same in the main branch, except they are in the original location.

Reply:
You are right, I confused with the other pyproject.toml.